Dockerfile.test

```diff
@@ -1,4 +1,4 @@
-FROM conjur
+ARG VERSION=latest
+FROM conjur:${VERSION}
```

Comment:
Oh, hey, can docker on our executor's handle this now? That'd be very exciting....

Reply:
Yeah, I had been wondering this about it since we upgraded Docker on the executors, and it works! I think we can simplify some of our bash/sed-fu that does the same thing in some of our projects.

Right now only ARG can come before FROM

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact